yarn-project/boxes/blank/src/index.ts

```diff
@@ -18,27 +18,25 @@ export const contractAbi: ContractAbi = BlankContractAbi;
 export const SANDBOX_URL: string = process.env.SANDBOX_URL || 'http://localhost:8080';
 export const pxe: PXE = createPXEClient(SANDBOX_URL);
 
-export const CONTRACT_ADDRESS_PARAM_NAMES = ['owner', 'contract_address', 'recipient'];
-export const FILTERED_FUNCTION_NAMES = [];
-
-export const DEFAULT_PUBLIC_ADDRESS: string = '0x25048e8c1b7dea68053d597ac2d920637c99523651edfb123d0632da785970d0';
-
 let contractAddress: string = '';
 
 // interaction with the buttons, but conditional check so node env can also import from this file
 if (typeof document !== 'undefined') {
   document.getElementById('deploy')?.addEventListener('click', async () => {
     contractAddress = await handleDeployClick();
+    // eslint-disable-next-line no-console
     console.log('Deploy Succeeded, contract deployed at', contractAddress);
   });
 
   document.getElementById('interact')?.addEventListener('click', async () => {
     const interactionResult = await handleInteractClick(contractAddress);
+    // eslint-disable-next-line no-console
     console.log('Interaction transaction succeeded', interactionResult);
   });
 }
 
 export async function handleDeployClick(): Promise<string> {
+  // eslint-disable-next-line no-console
```

Comment:
inclined to leave these in, so there's at least some sign things are working for the user's browser in the console

Reply:
yep, agree!